src/strong_branching.jl

```diff
@@ -79,7 +79,7 @@ function Bonobo.get_branching_variable(
                 Bonobo.get_branching_indices(tree.root),
             )
             status = check_feasibility(branching.bounded_lmo)
-            if status == OPTIMALS
```

Comment:
this was always returning false then??

Reply:
Rather, it threw an error because the compiler did not know the flag. 
In the MOI case, we have a separate strong branching function, which is the same as before.